meta-lxatac-bsp/recipes-bsp/barebox/barebox-tools.bbappend

```
@@ -0,0 +1 @@
+BAREBOX_TOOLS:append = "rk-usb-loader"
```

You're appending to a space-separated list without adding a space.
```suggestion
BAREBOX_TOOLS:append = " rk-usb-loader"
```